### PR TITLE
Api/articles/comments

### PR DESCRIPTION
**Pull request by erkina308**

### CODE REVIEW

File: controllers/articles.controllers.js [hunk 1]
```diff
@@ -10,3 +14,14 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getCommentByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  selectCommentByArticleId(article_id)
+    .then((response) => {
+      res.status(200).send(response);
```
Comment (nc-bela-bertalan) [inline controllers/articles.controllers.js:22]: Please send the comments back in an object on a key of 'comments'.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -60,8 +63,42 @@ describe("/api/articles/:article_id", () => {
...
+      });
+  });
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET: 200 responds with an array of comments for the given article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:77]: For proper response object, see comment in controller.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -60,8 +63,42 @@ describe("/api/articles/:article_id", () => {
...
+        expect(body.msg).toBe("Article does not exist");
+      });
+  });
+  test("GET: 400 sends an appropriate error status alongside an error message when given an invalid id", () => {
+    return request(app)
+      .get("/api/articles/invalid-id/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:102]: Please add a test where it responds with an empty array of comments and status 200.

**Review by nc-bela-bertalan — COMMENTED** [review]
Hey Erkin, nice work so far, please see my comments to refactor your code.
